### PR TITLE
Cris.shupp/5580 slack deploy notifications

### PR DESCRIPTION
**Pull request by cshupp1**
## Description of change

Adding in slack notifications for changes to benefits intake and forms.

## Original issue(s)
https://vajira.max.gov/browse/API-5580
https://vajira.max.gov/browse/API-5940

## Things to know about this PR

RSPEC testing added.

### CODE REVIEW

**Review by mviterna — DISMISSED** [review]

**Review by gpbowman-git — DISMISSED** [review]

**Comment by LindseySaari** [comment]
Can you add a description to the PR please?

**Comment by LindseySaari** [comment]
The migrations will have to be broken out as well

File: config/sidekiq_scheduler.yml [hunk 1]
```diff
@@ -215,6 +215,10 @@ Veteran::VSOReloader:
   class: Veteran::VSOReloader
   description: Pulls down current list of VSOs and updates accordingly 
 
+VBADocuments::DeploymentsNotifier:
+  at: "<%=0.seconds.from_now%>"
```
Comment (LindseySaari) [inline config/sidekiq_scheduler.yml:219]: What is the reason for 0.seconds.from_now?

File: config/initializers/sidekiq.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@
   config.super_fetch! if defined?(Sidekiq::Pro)
 
   config.on(:startup) do
-    Sidekiq.schedule = YAML.load_file(File.expand_path('../sidekiq_scheduler.yml', __dir__))
+    Sidekiq.schedule = YAML.safe_load(ERB.new(File.read(File.expand_path('../sidekiq_scheduler.yml', __dir__))).result)
```
Comment (LindseySaari) [inline config/initializers/sidekiq.rb:21]: Was this changed for development/testing purposes?

File: config/sidekiq_scheduler.yml (hunk 1 above)
Comment (cshupp1) [reply config/sidekiq_scheduler.yml:219]: Runs on startup.

File: config/initializers/sidekiq.rb (hunk 2 above)
Comment (cshupp1) [reply config/initializers/sidekiq.rb:21]: I don't understand your question

**Review by gpbowman-git — APPROVED** [review]

**Review by gpbowman-git — APPROVED** [review]

**Pull request merged**